src/topology/stone_cech.lean

```diff
@@ -172,7 +170,7 @@ lemma ultrafilter_extend_eq_iff {f : α → γ} {b : ultrafilter α} {c : γ} :
    -- the facts that ultrafilter.extend is a continuous extension of f.
    let b' : ultrafilter (ultrafilter α) := b.map pure,
    have t : b'.val ≤ 𝓝 b,
-     from ultrafilter_converges_iff.mpr (by exact (bind_pure _).symm),
+     from ultrafilter_converges_iff.mpr (by exact (_root_.bind_pure _).symm),
```

Comment:
Why is this necessary?  AFAICT there is only `bind_pure` and `filter.bind_pure`.  The `filter.bind_pure` is protected though so there shouldn't be any ambiguity.

Reply:
I guess I added this before making `filter.bind_pure` protected. I'll rename `filter.bind_pure` and remove this `_root_.`